packages/sdk/src/tests/controllers/LayoutController.test.ts

```diff
@@ -92,7 +94,21 @@ describe('LayoutController', () => {
         expect(mockedEditorApi.removeLayout).toHaveBeenCalledTimes(1);
         expect(mockedEditorApi.removeLayout).toHaveBeenCalledWith('1');
     });
-    it('Should be possible to create a layout', async () => {
+    it('Should be possible to create a layout with a preset', async () => {
+        const preset = {
+            name: 'name',
+            intent: LayoutIntent.print,
+            unit: MeasurementUnit.mm,
+            width: '100 mm',
+            height: '200 mm',
+        };
+
+        await mockedLayoutController.create('1', [preset]);
+        
+        expect(mockedEditorApi.addLayouts).toHaveBeenCalledTimes(1);
+        expect(mockedEditorApi.addLayouts).toHaveBeenCalledWith('1', JSON.stringify([preset]));
```

Comment:
Maybe manually add the serialized variant so that if someone updates it (by mistake) this test will break 😅 

```suggestion
        expect(mockedEditorApi.addLayouts).toHaveBeenCalledWith('1', JSON.stringify([preset]));
```